tests/test_ratio_of_means.py

```diff
@@ -73,16 +73,32 @@ def test_ratio_of_means(self, base_file_name):
             columns=["default_forward", "default_backward", "default_construction"]
         )
 
-        actual_output = ratio_of_means(
-            input_data,
-            target="question",
-            period="date",
-            reference="identifier",
-            strata="group",
-            auxiliary="other",
-            filters=filter_df,
-        )
-
+        if base_file_name not in ["19_link_columns", '28_link_columns_filtered']:
```

Comment:
Personally I would put this as a 
```{python}
 if base_file_name in ["19..."]:
          Handle exception
 else:
          Default method 
```
Just because its sort of difficult to read and follow which cases are the exception. If you needed to update the code for the exception, my initial thought would be to change lines 77-85. Happy to hear counter arguments to this though :)

Reply:
Pushed advised change